src/helpers/messages/get-time.ts

```diff
@@ -0,0 +1,5 @@
+export const getTime: (timestamp: number) => string = (timestamp: number) => {
```

Comment:
Export this one instead: https://github.com/ShokuApp/individual-forward/blob/a68cb7304a91f16aff8293066f08bc2b0ef5b7ef/src/components/messages/conversation-preview.tsx#L88-#L107